cache/solidity-files-cache.json

```diff
@@ -1,6 +1,38 @@
 {
   "_format": "hh-sol-cache-2",
   "files": {
```

Comment:
This folder should be part of the .gitignore right?

Reply:
Yes, definitely! Didn't realize they were included